packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -105,10 +85,9 @@ const BrandFooter = ({
   customProps.className);
 
   let processedSections;
-  if (links.length > 0 && sections.length === 0) {
+  if (sections.length === 0) {
```

Comment:
This logic was executing previously if links were provided but sections were not. It wouldn't have executed in the case where no links and no sections were provided. Now that links are removed entirely, is this block necessary at all?

Reply:
Removed it here https://github.com/cerner/terra-framework/pull/1167/commits/b7d34c6ed6202c60788979859d48f35354d4d505.